### PR TITLE
Address cleanup concern when kyverno tests fail

### PR DESCRIPTION
**Pull request by gparvin**
Cleanup can be a concern when the tests are run on a long running
cluster.  This change makes sure cleanup is called.

Refs:
 - https://github.com/stolostron/backlog/issues/22938

Signed-off-by: Gus Parvin <gparvin@redhat.com>

### CODE REVIEW

File: test/integration/policy_kyverno_generators_test.go [hunk 1]
```diff
@@ -34,7 +34,7 @@ var _ = Describe("GRC: [P1][Sev1][policy-grc] Test the kyverno generator policie
 	const kyvernoInstallURL = "https://raw.githubusercontent.com/stolostron/policy-collection/main/community/CM-Configuration-Management/policy-install-kyverno.yaml"
 	const kyvernoInstallPolicy = "policy-install-kyverno"
 
-	Describe("Test stable kyverno generator policies", func() {
+	Describe("Test stable kyverno generator policies", Ordered, func() {
```
Comment (dhaiducek) [inline test/integration/policy_kyverno_generators_test.go:37]: I think this `Describe` can be removed and the `Ordered` added to its parent. I'm not sure a nested Describe is necessary here.

**Review by dhaiducek — CHANGES_REQUESTED** [review]

**Review by JustinKuli — DISMISSED** [review]
/retest

Looks good! I think this will help prevent the situation our e2e cluster is currently in :D

**Comment by gparvin** [comment]
hibernation failures
/retest

**Comment by gparvin** [comment]
/retest
Maybe it will get a new cluster...

**Review by dhaiducek — APPROVED** [review]

**Comment by gparvin** [comment]
Looks like an unrelated error in limitrange test.
/retest

**Pull request merged**